### PR TITLE
[13.0][ADD] fleet_vehicle_license_plate_alt: Alternative license plate for vehicles

### PR DESCRIPTION
**Pull request by mamcode**
screenshot-localhost_8099-2020 05 21-11_17_43

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
pls squash your commits

File: fleet_vehicle_license_plate_alt/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Fleet Alternative License Plate",
+    "summary": "Add a new field in the vehicle, to store an alternative license plate.",
+    "category": "Human Resources/Fleet",
+    "author": "Druidoo, Odoo Community Association (OCA)",
+    "maintainers": ["mamcode", "ivantodorovich"],
+    "development_status": "Stable",
+    "website": "https://www.druidoo.io/",
```
Comment (ivantodorovich) [inline fleet_vehicle_license_plate_alt/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/fleet",
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by mamcode** [comment]
> pls squash your commits

Done.

**Review by ivantodorovich — APPROVED** [review]

File: fleet_vehicle_license_plate_alt/models/fleet_vehicle.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class FleetVehicle(models.Model):
+    _inherit = "fleet.vehicle"
+
+    license_plate_alt = fields.Char(
```
Comment (mymage) [inline fleet_vehicle_license_plate_alt/models/fleet_vehicle.py:11]:
```suggestion
    license_plate_alt = fields.Char("Alternative Licence Plate",
```

**Review by mymage — APPROVED** [review]
Only a suggestion to do label more readable.

**Comment by ivantodorovich** [comment]
seems pre-commit is failing on that last commit.
In any case, can you pls squash them ?

**Comment by mamcode** [comment]
> seems pre-commit is failing on that last commit.
> In any case, can you pls squash them ?

Done :)

**Pull request merged**